--- a/xbmc/settings/dialogs/GUIDialogAudioSettings.cpp
+++ b/xbmc/settings/dialogs/GUIDialogAudioSettings.cpp
@@ -198,7 +198,7 @@ void CGUIDialogAudioDSPSettings::CreateSettings()
       AddSpin(AUDIO_SETTING_CONTENT_TYPE, 15021, (int*)&CMediaSettings::Get().GetCurrentAudioSettings().m_MasterStreamTypeSel, modeEntries);
     }
 
-    CMediaSettings::Get().GetCurrentAudioSettings().m_MasterStreamType    = m_streamTypeUsed;
+    CMediaSettings::Get().GetCurrentAudioSettings().m_MasterStreamType = m_streamTypeUsed;
     SET_CONTROL_LABEL(CONTROL_TYPE_LABEL, g_localizeStrings.Get(CActiveAEDSP::Get().GetDetectedStreamType(m_ActiveStreamId)+INPUT_TYPE_LABEL_START));
 
     m_AddonMasterModeSetupPresent = false;
@@ -229,7 +229,7 @@ void CGUIDialogAudioDSPSettings::CreateSettings()
     AddSeparator(AUDIO_SEPERATOR_1);
 
     m_volume = g_application.GetVolume(false);
-    AddSlider(AUDIO_SETTINGS_VOLUME, 13376, &m_volume, VOLUME_MINIMUM, VOLUME_MAXIMUM / 100.0f, VOLUME_MAXIMUM, PercentAsDecibel, !g_application.m_pPlayer->IsPassthrough());
+    AddSlider(AUDIO_SETTINGS_VOLUME, 13376, &m_volume, VOLUME_MINIMUM, VOLUME_MAXIMUM / 100.0f, VOLUME_MAXIMUM, PercentAsDecibel, false);
     if (SupportsAudioFeature(IPC_AUD_AMP))
     {
       AddSlider(AUDIO_SETTINGS_VOLUME_AMPLIFICATION,
@@ -237,7 +237,7 @@ void CGUIDialogAudioDSPSettings::CreateSettings()
                 &CMediaSettings::Get().GetCurrentVideoSettings().m_VolumeAmplification,
                 VOLUME_DRC_MINIMUM * 0.01f, (VOLUME_DRC_MAXIMUM - VOLUME_DRC_MINIMUM) / 6000.0f, VOLUME_DRC_MAXIMUM * 0.01f,
                 FormatDecibel,
-                !g_application.m_pPlayer->IsPassthrough());
+                false);
       AddSeparator(AUDIO_SEPERATOR_2);
     }
 
